modules/programs/wezterm.nix

```diff
@@ -0,0 +1,617 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.programs.wezterm;
+
+  genericType = with types; oneOf [ str bool int float ];
+
+  boolToStr = boolVal: if boolVal then "true" else "false";
+
+  formatBindings = bindings:
+    pipe bindings [
+      (x: builtins.concatStringsSep "\n" x)
+      (x: lib.strings.removeSuffix "\n" x)
+      (x: builtins.replaceStrings [ "\n  \n" ] [ "\n" ] x)
+      (x: builtins.replaceStrings [ "\n" ] [ "\n    " ] x)
+    ];
```

Comment:
I agree it may be slightly unneeded, majority of the people who I've talked to that want to use it said they would prefer if it was formatted so they could read the generated code. If you feel like it should be removed then I can remove it.